searchlib/src/vespa/searchlib/features/random_normal_stable_feature.cpp

```diff
@@ -0,0 +1,79 @@
+// Copyright 2017 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+#include "random_normal_stable_feature.h"
+#include "utils.h"
+#include <vespa/searchlib/fef/properties.h>
+#include <vespa/fastos/time.h>
+
+#include <vespa/log/log.h>
+LOG_SETUP(".features.randomnormalstablefeature");
+
+namespace search {
+namespace features {
+
+RandomNormalStableExecutor::RandomNormalStableExecutor(uint64_t seed, double mean, double stddev) :
+    search::fef::FeatureExecutor(),
+    _rnd(mean, stddev, true),
```

Comment:
Shouldn't useSpare be false? We are re-seeding on each call to execute and need a new calculation when calling next() on RandomNormal.